### ISSUE TITLE
Add a [lang] attribute to the html element

### ISSUE THREAD

**Issue by jamesjlyons** [liferay-design/liferay.design#267]
If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.

[More info on lang attributes](https://dequeuniversity.com/rules/axe/3.1/html-has-lang)

**Comment by plhnk** [comment]
[fixed](https://github.com/liferay-design/liferay.design/commit/ccb0ae7f23ed0d42df3ffb375eaae1984c1e1615)
